src/platform/site-wide/va-footer/containers/Main.jsx

```diff
@@ -88,7 +80,7 @@ export class Main extends React.Component {
           </h4>
         </li>
         <li>
-          <a role="button" tabIndex="0" onClick={this.openModal}  onKeyPress={this.handleKeyPress} >Veterans Crisis Line</a>
+          <button className="va-button-link va-overlay-trigger" data-show="#modal-crisisline">Veterans Crisis Line</button>
```

Comment:
If we can add `onClick={() => recordEvent({ event: FOOTER_EVENTS.CRISIS_LINE })}` here to continue capturing the Crisis Line event in the footer, that should resolve the conflicts.